### DISCUSSION TITLE
Insel Umfahrung

### DISCUSSION THREAD

**Discussion by PhilippS1** [Courseplay/Courseplay_FS22#2318] [Q&A] [answered]
Hallo zusammen,

wie groß muß eine Feldinsel sein damit diese beim Coursegenerieren erkannt wird?
Hintergrund zu dieser Frage ist, ich spiele auf der Karte Ostseeküste. Die Karte hat einige Strommasten sowie Gullideckel auf den Feldern, diese wer jedoch beim generieren nicht immer erkannt.

Gruß

**Comment by Tensuko** [comment, marked as answer]
Größer als das was da ist.
Male doch mit Gras einen kleinen Kreis um die Objekte, generiere einen Kurs und guck ob es reicht.
Sonst male den ein Stück größer und versuch noch mal.
